wrapper/src/main/java/software/amazon/jdbc/plugin/AwsSecretsManagerConnectionPlugin.java

```diff
@@ -137,16 +140,6 @@ public Connection connect(
       final JdbcCallable<Connection, SQLException> connectFunc)
       throws SQLException {
 
-    if (this.secretsManagerClient == null || this.getSecretValueRequest == null) {
-      this.secretsManagerClient = SecretsManagerClient.builder()
-          .credentialsProvider(AwsCredentialsManager.getProvider(hostSpec, props))
```

Comment:
This line was added to support the custom credentials request that was recently merged in, I think we'll have to add it to the default secretsManagerClientFunc. It ensures that customers have a way to set their own credential provider

Reply:
Good catch